pkgs/top-level/all-packages.nix

```
@@ -11200,9 +11200,7 @@ with pkgs;
 
   patchage = callPackage ../applications/audio/patchage { };
 
-  patchance = python3Packages.callPackage ../applications/audio/patchance {
-    inherit (qt5) qttools;
-  };
+  patchance = python3Packages.callPackage ../applications/audio/patchance { };
```

```suggestion
  patchance = python3Packages.callPackage ../applications/audio/patchance {
    inherit (qt5) qttools wrapQtAppsHook;
  };
```
I think it is a good idea to add `wrapQtAppsHook` here.

---

I'm inclined to leave this the way it is, but I'm curious, what is your reason for wanting to put this in `all-packages.nix`? Wouldn't it be best to keep it in the main package file so everything is consolidated in one place? I'm interested because I've seen it done both ways and there doesn't seem to be a "best practice" for it.